app/core/redis.py

```diff
@@ -1,20 +1,39 @@
 import redis
 from loguru import logger
 import json
+from os import getenv
+from dotenv import load_dotenv
 from app.stac.services.stac_item_creator import STACItemCreator
 
+load_dotenv(".env")
 
-def create_redis_connection(host: str, port: int, db: int):
-    return redis.Redis(host=host, port=port, db=db)
+REDIS_HOST = getenv("REDIS_HOST")
+REDIS_PORT = int(getenv("REDIS_PORT"))
+REDIS_DB = int(getenv("REDIS_DB", 0))
+REDIS_OUTGOING_LIST_NAME = getenv("REDIS_OUTGOING_LIST_NAME")
+
+
+def create_redis_connection():
+    return redis.Redis(host=REDIS_HOST, port=REDIS_PORT, db=REDIS_DB)
```

Comment:
not using REDIS_DB in the previous services, and our redis server is not shared with other projects